packages/grafana-ui/src/components/index.ts

```diff
@@ -161,6 +160,7 @@ const LegacyForms = {
   NoOptionsMessage,
   ButtonSelect,
   Input,
+  Switch,
 };
-
+export { Switch };
```

Comment:
Should we export this one both in the root and within the LegacyForms?

Reply:
Yes, with previous migrations I had the issue that the Enterprise build broke. I'll change the root export when using the new form styles, so there is always a `Switch` exported in root.